css/basic/base.css

```diff
@@ -0,0 +1,41 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap');
```

Comment:
Gal reikia daugiau weights? Bent jau normal ir bold? Gal dar koki thin. Siaip nezinau kaip tiksliai veikia ar reikia includinti ar ne 😀 ar cia mums 600 to uzteks?

Reply:
Na as paliginau, lyg tais ir nieko, bet toj paziuresiu del viso pikto ir pridesiu

Reply:
Done.